client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -2,18 +2,99 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import { getCurrentUser } from 'state/current-user/selectors';
 
 export class CommentDetailReply extends Component {
+	state = {
+		commentText: '',
+		haveFocus: false,
+	};
+
+	handleBlur = () => this.setState( { haveFocus: false } );
+
+	handleFocus = () => this.setState( { haveFocus: true } );
+
+	handleKeyDown = event => {
```

Comment:
It might be a bit overkill, but let's try using the keyboard-shortcuts lib. We can make a new section in `key-binding.js`. I can see us wanting to add other shortcuts for navigating comments.